localstack/services/awslambda/invocation/lambda_models.py

```diff
@@ -64,6 +64,9 @@
 SNAP_START_SUPPORTED_RUNTIMES = [Runtime.java11, Runtime.java17]
 
 
+BUCKET_ACCOUNT = "949334387222"
```

Comment:
"Hiding" LocalStack-internal lambda buckets basically addresses this question https://github.com/localstack/localstack/issues/8494